conda/plan.py

```diff
@@ -483,7 +486,9 @@ def remove_actions(prefix, specs, index, force=False, pinned=True):
             if any(ms.name == 'conda' for ms in mss):
                 sys.exit("Error: 'conda' cannot be removed from the root environment")
             else:
-                sys.exit("Error: this 'remove' command cannot be executed because it\nwould require removing 'conda' dependencies.")
+                msg = "Error: this 'remove' command cannot be executed because it"
+                msg += "\nwould require removing 'conda' dependencies."
```

Comment:
Same comment as above with the string concat.  Also prefer `\n` being *actually* at the end of a line if it's going to be used.

    msg = ("Error: this 'remove' command cannot be executed because it\n"
           "would require removing 'conda' dependencies.")